lib/Connection.js

```diff
@@ -48,6 +49,7 @@ const availableTransports = {
  * @property {string} [ipc=*NA] - The key name/security route to XMLSERVICE job. Default is ``*NA``.
  * @property {string} [ctl=*here] - The control options for XMLSERVICE jobs. Default is ``*here``.
  * @property {string} [xslib=QXMLSERV] - The XMLSERVICE library. Default is ``QXMLSERV``.
+ * @property {object} odbcConnection - An optional odbc connection instead of a creating a new one.
```

Comment:
Line 52 documents the idb transport options therefore we should not add odbcConnection property here.

```suggestion

```